src/pages/commits/ActiveCommits/CommitChartLabel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
As of React 17 we actually don't need to import React anymore. https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html